packages/ui/src/components/CodeBlock/CodeBlock.tsx

```diff
@@ -177,6 +196,11 @@ export const CodeBlock = ({
               paddingBottom: '4px',
             }}
             renderer={renderer}
+            contentEditable={focusable}
+            onBeforeInput={(e: any) => {
```

Comment:
a bit hacky but its the only way i've found to make the codeblock readonly and focusable (so that ⌘ A only selects content in the codeblock)